engines/hadesch/rooms/ferry.cpp

```diff
@@ -287,7 +287,7 @@ static const struct {
 	{ "V9250tD0", { "V9250aD0", _hs("Charon must sit next to smoking beings") } },
 	{ "V9260tA0", { "V9260nA0", _hs("Charon can't have any monsters at the head of the boat") } },
 	{ "V9260tB0", { "V9260nB0", _hs("No animals at the head of the boat") } }, // 5
-	{ "V9260tC0", { "V9260nC0", _hs("Charon wont sit next to two-headed beings") } },
+	{ "V9260tC0", { "V9260nC0", _hs("Charon won't sit next to two-headed beings") } },
```

Comment:
This should be a FIXME comment instead, as it modifies data

Reply:
Yes, i thought so as well, but look at the following lines, they all have won't with an apostrophe, so i figured it would be safe to change this single line?

Please advise

Reply:
No, please, refrain from changing game data, no matter the patterns you see.

Reply:
reverted to a fixme

thank you

Reply:
what did i break now? 😞

sorry

Reply:
@raziel- you didn't break anything. Builds have been broken by another commit